test/libsolidity/SemanticTest.cpp

```diff
@@ -47,6 +53,25 @@ SemanticTest::SemanticTest(string const& _filename, langutil::EVMVersion _evmVer
 	m_lineOffset(m_reader.lineNumber()),
 	m_enforceViaYul(enforceViaYul)
 {
+	using namespace std::placeholders;
+	// clang-format off
+	auto simpleSmokeBuiltin = std::make_shared<Builtin>(
+		std::bind(&SemanticTest::builtinSmokeTest, this, _1)
+	);
+	m_builtins =
+		{
+			{"smoke",
+				{{"test0", simpleSmokeBuiltin},
+			 	{"test1", simpleSmokeBuiltin},
+			 	{"test2", simpleSmokeBuiltin}}
+			}
+		};
```

Comment:
Maybe this looks better like this?
```suggestion
	m_builtins = {
		{
			"smoke", {
				{"test0", simpleSmokeBuiltin},
				{"test1", simpleSmokeBuiltin},
				{"test2", simpleSmokeBuiltin}
			}
		}
	};
```